### ISSUE TITLE
Cross-fused renal ectopia subtypes

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7197]
Should we add the following five subtypes opf Crossed fused [renal ectopia HP:0004736](https://hpo.jax.org/app/browse/term/HP:0004736)?
ALso, add synonym with dash (Cross-fused renal ectopia)

Cross-fused renal ectopia is the second most common fusion anomaly, with a fre- quency of 1/7500 births [22]. This condi- tion refers to fusion of the two kidneys on the same side of the spine. The ureter of the ectopic kidney crosses the midline and joins the urinary bladder at the orthotopic site. Depending on the fusion site and final renal shape, different variants of crossed- fused renal ectopia are described in the lit- erature, including type A (inferior crossed fusion), type B (sigmoid kidney), type C, (lump kidney), type D, (disc kidney), type E, (L-shaped kidney), and type F (superior- ly crossed-fused).

See https://www.ajronline.org/doi/pdf/10.2214/AJR.17.18371

**Comment by patt1morgan** [comment]
@pnrobinson 
Hi Peter, 
This may be a question to pose to your renal 'specialists'. Prenatally, we can identify if there may be a cross-fused kidney, but we are not experts in the various types of fusion.

**Comment by pnrobinson** [comment]
added subtypes.
